### ISSUE TITLE
[WARNING] Prisma Client 2.10 bug

### ISSUE THREAD

**Issue by Stevenje** [chagadev/fullstack-boilerplate#16]
There seems to be a issue after upgrading to Prisma Client 2.10 and nexus.

I'm having a really hard time reverting the Client to 2.9. Even after reverting back all Package.json to 2.9 and resetting yarn.lock it keeps generating the client using 2.10. 

```
TypeError: xs.reduce is not a function
    at Object.exports.indexBy (/Users/steven/Documents/boilerplate/versiontwo/node_modules/nexus-plugin-prisma/src/utils.ts:66:15)
    at new DmmfDocument (/Users/steven/Documents/boilerplate/versiontwo/node_modules/nexus-plugin-prisma/src/dmmf/DmmfDocument.ts:30:26)
    at Object.exports.getTransformedDmmf (/Users/steven/Documents/boilerplate/versiontwo/node_modules/nexus-plugin-prisma/src/dmmf/transformer.ts:16:20)
    at new SchemaBuilder (/Users/steven/Documents/boilerplate/versiontwo/node_modules/nexus-plugin-prisma/src/builder.ts:256:7)
    at Object.build (/Users/steven/Documents/boilerplate/versiontwo/node_modules/nexus-plugin-prisma/src/builder.ts:175:19)
    at Object.onInstall (/Users/steven/Documents/boilerplate/versiontwo/node_modules/nexus-plugin-prisma/src/plugin.ts:46:62)
    at /Users/steven/Documents/boilerplate/versiontwo/node_modules/@nexus/schema/src/builder.ts:657:44
    at Array.forEach (<anonymous>)
    at SchemaBuilder.beforeWalkTypes (/Users/steven/Documents/boilerplate/versiontwo/node_modules/@nexus/schema/src/builder.ts:651:18)
    at SchemaBuilder.getFinalTypeMap (/Users/steven/Documents/boilerplate/versiontwo/node_modules/@nexus/schema/src/builder.ts:741:10)
[ERROR] 16:55:38 TypeError: xs.reduce is not a function

Watching... /Users/steven/Documents/boilerplate/versiontwo/packages/prisma/schema.prisma

✔ Generated Prisma Client (version: 2.10.0) to ./../../node_modules/@prisma/client in 430ms


  Dev server running at:
  > Local:    http://localhost:3000/
  > Network:  http://192.168.2.135:3000/
```

**Comment by chagadev** [comment]
Fixed by updating nexus-plugin-prisma@0.23.0

**Comment by chagadev** [comment]
There are warnings when building Nexus schema concerning @nexus/schema@0.17.0 and graphql@15.4.0 but it seems to work (we'll reopen this issue if we encounter problems)
